share/completions/iftop.fish

```diff
@@ -0,0 +1,29 @@
+function __fish_iftop_sort_orders
+	echo -e "2s\tSort by 2s traffic average"
+	echo -e "10s\tSort by 10s traffic average (default)"
+	echo -e "40s\tSort by 40s traffic average"
+	echo -e "source\tSort by source address"
+	echo -e "destination\tSort by destination address"
+end
+
+complete -c iftop -f
+
+complete -c iftop -s h -d "Print a summary of usage"
+complete -c iftop -s n -d "Don't do hostname lookups"
+complete -c iftop -s N -d "Don't resolve port number to service names"
+complete -c iftop -s p -d "Run in promiscuous mode"
+complete -c iftop -s P -d "Turn on port display"
+complete -c iftop -s l -d "Include link-local IPv6 addresses"
+complete -c iftop -s b -d "Don't display bar graphs of traffic"
+complete -c iftop -s m -x -d "Set the upper limit for the bandwidth scale"
+complete -c iftop -s B -d "Display bandwidth rates in bytes/sec rather than bits/sec"
```

Comment:
As an idea for enhancing completion for this option: provide 'K', 'M' or 'G' suffixes when there is some number entered already as the current token. As for the code, the completion improvement can be based on [mine](https://github.com/fish-shell/fish-shell/pull/10757/files#diff-25723ae18a81f6705d4abcdf09095a75f106e6b05fc827fbff5d99c7a20a2a2fR3) can be used:

```fish
# Usage: __fish_complete_extensions extension1 extension2 ...
# Example: __fish_complete_extensions png jpg
function __fish_complete_extensions \
    --description 'Suggest provided extensions as $argv if the current token ends with a dot'

    set token (commandline -c -t)

    switch "$token"
        case '*.'
            for extension in $argv
                if string match --quiet --regex '\\\t|\t' -- $extension
                    echo -e "$token$extension"
                else
                    printf "%s%s\n" $token $extension
                end
            end
    end
end
```